<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingPage</class>
 <widget class="QWidget" name="SettingPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_ScheduleName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>计划名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ScheduleName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_SourceDir">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>备份源目录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SourceDir">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_SaveDir">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>备份保存目录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SaveDir">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_ChooseSourceDir">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择源目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_ChooseSaveDir">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择保存目录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_CompressType">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择压缩格式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_CompressType">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>zip</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7z</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tar.gz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tar.bz2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>tar.xz</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox_RestrictSaveTime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>限制备份保存时间</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_RestrictSaveDirSize">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>限制备份保存目录大小</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_RestrictTime">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择限制时间</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_RestrictTime">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>1 天</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 天</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 天</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7 天</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_RestrictSize">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择限制大小</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_RestrictSize">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_RestrictSizeUnit">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>MB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GB</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_TriggerType">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择触发方式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_TriggerType">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>当到达定时时间</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>当文件发生修改</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_InterveralTime">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择定时备份时间</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_InterveralTime">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="value">
    <number>30</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_InterveralTimeUnit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>分钟</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>小时</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Button_Confirm">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_Cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
